config/webpack.config.js

```diff
@@ -4,8 +4,8 @@ const ExtractTextPlugin = require('extract-text-webpack-plugin');
 const ManifestPlugin = require('webpack-manifest-plugin');
 const ChunkManifestPlugin = require('chunk-manifest-webpack-plugin');
 const WebpackMd5Hash = require('webpack-md5-hash');
-const bourbon = require('bourbon').includePaths;
-const neat = require('bourbon-neat').includePaths;
+// const bourbon = require('bourbon').includePaths;
+// const neat = require('bourbon-neat').includePaths;
```

Comment:
Yeah I believe that it resolves the same problem you have linked. With the previous version of USWDS, it would install bourbon and neat in the top-level of our node_modules folder, so we could require it this way. But with the upgrade, it isn't top-level anymore. Not exactly sure how that works, but I'm finding that to be the case. 

Our SASS style includes Bourbon and Neat from USWDS's lib directory (see the top of style.scss) so I don't think that costs us any anything - we still have the grid and all of USWDS style. I'm not exactly sure what these do in our Webpack but maybe @jbalboni knows for sure. 

If we find these to be important for one reason or another we can just add bourbon and neat to our `package.json` and uncomment those lines. I didn't do that because USWDS is using such specific [versions](https://github.com/18F/web-design-standards/blob/develop/package.json#L74).

Reply:
As far as I know, these don't really do anything right now. They allow us to import directly from bourbon/neat, I think, but we don't do that.

Reply:
Sweet! So maybe we can leave it commented out for now, and install them later if we decide that we need them as direct imports instead of from USWDS's install folder.

Reply:
Thanks @ncksllvn @jbalboni.